slick/src/main/scala/slick/jdbc/JdbcBackend.scala

```diff
@@ -172,54 +172,92 @@ trait JdbcBackend extends RelationalBackend {
-      *     connection instead).</li>
-      *   <li>`maxConnections` (Int, optional, default: `numThreads` * 5): The maximum number of
-      *     connections in the pool.</li>
-      *   <li>`minConnections` (Int, optional, default: same as `numThreads`): The minimum number
-      *     of connections to keep in the pool.</li>
-      *   <li>`connectionTimeout` (Duration, optional, default: 1s): The maximum time to wait
-      *     before a call to getConnection is timed out. If this time is exceeded without a
-      *     connection becoming available, a SQLException will be thrown. 1000ms is the minimum
-      *     value.</li>
-      *   <li>`validationTimeout` (Duration, optional, default: 1s): The maximum amount of time
-      *     that a connection will be tested for aliveness. 1000ms is the minimum value.</li>
-      *   <li>`idleTimeout` (Duration, optional, default: 10min): The maximum amount
-      *     of time that a connection is allowed to sit idle in the pool. A value of 0 means that
-      *     idle connections are never removed from the pool.</li>
-      *   <li>`maxLifetime` (Duration, optional, default: 30min): The maximum lifetime of a
-      *     connection in the pool. When an idle connection reaches this timeout, even if recently
-      *     used, it will be retired from the pool. A value of 0 indicates no maximum
-      *     lifetime.</li>
-      *   <li>`connectionInitSql` (String, optional): A SQL statement that will be
-      *     executed after every new connection creation before adding it to the pool. If this SQL
-      *     is not valid or throws an exception, it will be treated as a connection failure and the
-      *     standard retry logic will be followed.</li>
-      *   <li>`initializationFailFast` (Boolean, optional, default: false): Controls whether the
-      *     pool will "fail fast" if the pool cannot be seeded with initial connections
-      *     successfully. If connections cannot be created at pool startup time, a RuntimeException
-      *     will be thrown. This property has no effect if `minConnections` is 0.</li>
-      *   <li>`leakDetectionThreshold` (Duration, optional, default: 0): The amount of time that a
-      *     connection can be out of the pool before a message is logged indicating a possible
-      *     connection leak. A value of 0 means leak detection is disabled. Lowest acceptable value
-      *     for enabling leak detection is 10s.</li>
-      *   <li>`connectionTestQuery` (String, optional): A statement that will be executed just
-      *     before a connection is obtained from the pool to validate that the connection to the
-      *     database is still alive. It is database dependent and should be a query that takes very
-      *     little processing by the database (e.g. "VALUES 1"). When not set, the JDBC4
-      *     `Connection.isValid()` method is used instead (which is usually preferable).</li>
+      *   <li>Essentials:
```

Comment:
Changes the docs organization and add docs to the HikariCP configurations added. But most of the content is untouched.